src/app/features/add-course/add-course-window/add-course-window.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="add-course">
+  <h1 class="add-course__title">New Course</h1>
+
+  <form [formGroup]="courseForm" #registerForm="ngForm" (ngSubmit)="submitCourse" class="add-course__form">
```

Comment:
(ngSubmit)="submitCourse" doesn't work. Use (ngSubmit)="submitCourse()" instead

Reply:
I was inattentive, thank you!